test/parallel/test-trace-event.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const assert = require('assert');
+const cp = require('child_process');
+const common = require('../common.js');
+const fs = require('fs');
+
+const CODE = 'for (var i = 0; i < 100000; i++) { \"test\" + i }';
+const FILE_NAME = 'node_trace.1.log';
+
+common.refreshTmpDir();
+process.chdir(common.tmpDir);
+
+const proc = cp.spawn(process.execPath,
+  [ '--enable-tracing', '-e', CODE ]);
+
+proc.once('exit', common.mustCall(() => {
+  assert(common.fileExists(FILE_NAME));
+  fs.readFile(FILE_NAME, (err, data) => {
+    fs.unlinkSync(FILE_NAME);
```

Comment:
Tiniest of nits that can totally be ignored: No need to clean up after the test in the temp directory. Each test that needs to use the temp directory runs `common.refreshTmpDir()` which will clean up anything there.